go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	github.com/biogo/store v0.0.0-20160505134755-913427a1d5e8
 	github.com/cenkalti/backoff v2.1.1+incompatible
 	github.com/client9/misspell v0.3.4
-	github.com/cockroachdb/apd v1.1.0 // indirect
+	github.com/cockroachdb/apd v1.1.0
```

Comment:
not sure why this became a direct dependency...

Reply:
make buildshort again. You have leftover generated files that import apd